autoload/ale/cursor.vim

```diff
@@ -130,8 +146,16 @@ function! ale#cursor#ShowCursorDetail() abort
 
     if !empty(l:loc)
```

Comment:
Take this part of the function here, and move it to a script local function which accepts the arguments `(loc, stay_here)` Then you can call that function instead of this one. You can avoid all of the logic above for showing detailed information automatically, including searching for the item again.